packages/components/src/molecules/SearchProducts/SearchProductItemControl.tsx

```diff
@@ -0,0 +1,127 @@
+import React, {  forwardRef, HTMLAttributes, useCallback, useState } from 'react'
+import { Badge, Icon, IconButton, Input, Loader, QuantitySelector } from '../..'
+
+import type { ReactNode, MouseEvent } from 'react'
+
+type StatusButtonAddToCartType = 'default' | 'inProgress' | 'completed'
+
+export interface SearchProductItemControlProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'children' | 'onClick'> {
+  children: ReactNode
+  availability: boolean
+  hasVariants: boolean
+  skuMatrixControl: ReactNode
+  quantity: number
+  onClick?(e: MouseEvent<HTMLButtonElement>): void
+  onChangeQuantity(value: number): void
+}
+
+const SearchProductItemControl = forwardRef<
+  HTMLDivElement,
+  SearchProductItemControlProps
+>(function SearchProductItemControl(
+  {
+    availability,
+    children,
+    hasVariants,
+    skuMatrixControl,
+    quantity,
+    onClick,
+    onChangeQuantity,
+    ...otherProps
+  },
+  ref
+) {
+  const [statusAddToCart, setStatusAddToCart] =
+    useState<StatusButtonAddToCartType>('default')
+  function stopPropagationClick(e: MouseEvent) {
```

Comment:
**question**: Should the product item be clickable in the list? (keeping the previous behaviour, click and go to the product page)